BookingSpecTests/RegistrationFeature.feature

```diff
@@ -0,0 +1,93 @@
+﻿Feature: Registration
+
+Scenario: Input valid credentials in order to Register
+	When I open browser
+	And I navigate to url "https://booking.com"
+	And I click Register button on Ribbon menu
+	And I set following parameters on Register Pop Up dialog
+	| Field    | Value                      |
+	| Email    | testmirantistest@gmail.com |
+	| Password | 1234qweR                   |
+	And I click Get Started button on PopUp
+	Then I see that I am Registered and Signed In
+
+Scenario Template: Attempt to Register using Invalid combinations of email and password
+	When I open browser
+	And I navigate to url "https://booking.com"
+	And I click Register button on Ribbon menu
+	And I set following parameters on Register Pop Up dialog
+		| Field    | Value      |
+		| Email    | <Email>    |
+		| Password | <Password> |
+	And I click Get Started button on PopUp
+	Then I wait while page popup is working
+	And I see message error with "<Message>"
+
+	Scenarios: 
+		| Email        | Password  | Message                                         |
+		| 2131312@.com | 1         | Password needs to be at least 8 characters long |
+		| 2131312@.com |           | Please add a password                           |
+		| 2131312@.com | wrongpass | Please enter a valid email address.             |
+		| 2131312.com  | 12341234  | Please enter a valid email address.             |
+		| 2131312@com  | 12341234  | Please enter a valid email address.             |
+		|              |           | Please enter a valid email address.             |
+
+Scenario: Input email and pass that already registered and make sure that I am signed in
```

Comment:
Let's rename it to something more simple like: Try to register as already registered user.